index.html

```diff
@@ -0,0 +1,1117 @@
+              to the Kentucky
+              senator's
+              Twitter account.</p>
+          </div>
+        </section>
+        <section id="top-stories" class="d-none d-md-block d-lg-block col-md-7 col-lg-8">
+          <h2>top Story</h2>
+          <div>
+            <img
+              src="https://d.newsweek.com/en/full/1575696/donald-trump.jpg?w=591&h=364&f=c4a7be2fe03204b5fb5327dde00d07c9"
+              alt="" class="w-100">
+            <h3>Trump Announces Activation of the National Guard in Worst-Hit COVID-19 States</h3>
+            <p>The federal government has deployed hundreds of tons of supplies from our national stockpile to
+              locations
+              with the greatest need in order to assist in those areas," the president said Sunday.</p>
+          </div>
+          <section id="culture-travel" class="row">
+            <h2 class="col-12">CULTURE & TRAVEL</h2>
+            <div class="col-md-4 col-lg-4">
+              <img
+                src="https://d.newsweek.com/en/full/1574874/spanish-steps-italy.jpg?w=397&h=397&f=ad0f7bdfd3dc9fffc99ae723d926ac32"
+                alt="" class="w-100">
+              <h4>Social Distancing Around the World in Photos</h4>
+              <p>From Rome's desolate Spanish Steps to New York's deserted Times Square, take a look at some of the
+                world's
+                busiest landmarks that are all now but eerily empty.</p>
+            </div>
+            <div class="col-md-4 col-lg-4">
+              <img
+                src="https://d.newsweek.com/en/full/1575422/breakfast-lunch-dinner.webp?w=397&h=397&f=a6dbd65ad3ba41063a6b46b1f92b7a60"
+                alt="" class="w-100">
+              <h4>Wanderlust Stricken? The 10 Best Travel TV Shows to Watch Now</h4>
+              <p>Travelers can still explore the world while binge-watching these travel TV shows in home isolation.
+                Thanks
+                to streaming services, we can virtually send ourselves to nearly any and every part of the planet in
+                just a
+                few clicks.</p>
+            </div>
+            <div class="col-md-4 col-lg-4">
+              <img
+                src="https://d.newsweek.com/en/full/1574573/taj-mahal.webp?w=397&h=397&f=6059c9040fd468ee41988930ca76f210"
+                alt="" class="w-100">
+              <h4>You Can Still See the World With These Virtual Tours</h4>
+              <p>What to do with the free time available when trips to the movies, a museum or the theater are out of
+                the
+                question? Here is a list of museums and other popular tourist attractions that have entire collections
+                or
+                performances available online.</p>
+            </div>
+          </section>
+          <section id="more-stories" class="row">
+            <h2 class="col-lg-12">more stories</h2>
+            <div class="col-lg-12 d-flex list-item mb-md-4">
+              <img
```

Comment:
Check the spacing on this section. Your articles are more separated than the original page.

On mobile screens, your section has no title and it looks disorganized.
![Screenshot from 2020-04-13 08-30-51](https://user-images.githubusercontent.com/51926937/79124078-2ee0b700-7d61-11ea-8b00-adaf592885bf.png)

Original:
![Screenshot from 2020-04-13 08-30-45](https://user-images.githubusercontent.com/51926937/79124085-330cd480-7d61-11ea-90c2-a59701a0ba3c.png)